src/app/lib/Cookie.js

```diff
@@ -1,16 +1,21 @@
 import Cookies from 'js-cookie';
 import jwt from 'jsonwebtoken';
+import allGroups from './groups.json';
+
+const isInValidGroup = function(userGroups) {
```

Comment:
agreed - I would normally use the first but did this for consistency

Reply:
Is ```const isInValidGroup = function(userGroups) {``` not just alternative syntax for creating a named function?

Reply:
Essentially yes, but this is JavaScript so of course they behave in subtly different ways that are unexpected 😅

For example, this is valid...
```
doAllTheThings();
function doAllTheThings() { ... }
```

but this is not...
```
doAllTheThings();
const doAllTheThings = function() { ... };
```

because functions get hoisted, but `const` and `let` do not.